### PR TITLE
Input and image component

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
1. Responsive Images.
2. input:
     a. Textbox.
     b. Validation and error types.

Image site link: https://ym-cl-input-and-image.netlify.app/components/responsive-image/image.html 
input site link: https://ym-cl-input-and-image.netlify.app/components/input/input.html

### CODE REVIEW

File: components/responsive-image/image.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<body>
+    <h1>Responsive Image</h1>
+    <p>By using below mentioned css classes, you can mirror the below components. </p>
+    
+    <!-- Responsive Image -->
+    <div class="container">
+        <!-- Image -->
+        <div class="image-container">
+            <img class="image-resp" src="../../images/face.jpg" alt="responsive image">
+        </div>
```
Comment (rahulk31) [inline components/responsive-image/image.html:20]: The image is not responsive, it does not fits the container when the size changes.
Example: I randomly updated the size of container to 500px and the image size did not change accordingly.

image

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    <h3>Input textbox:</h3>
+    <div class="container">
+        <!-- Input textbox -->
+        <div class="input-container">
+            <form class="input-flex">
+                <label for="name">First Name</label>
+                <input class="input" type="text" id="name" name="name">
+                <label for="email">Email Id<span class="input-required">*</span></label>
+                <input class="input" type="text" id="email" name="email" required>
```
Comment (rahulk31) [inline components/input/input.html]: For the email type inputs, set the type attribute as "email" to setup the browser level validation.

**Review by rahulk31 — COMMENTED** [review]
Overall looks good to me.

File: components/input/input.html (hunk 2 above)
Comment (yaswanthmyneni) [reply components/input/input.html]: ok

**Comment by iamalhera** [comment]
looks good, just make your containers width small, don't make it 100% of the viewport maybe 75% will look better

File: components/responsive-image/image.html (hunk 1 above)
Comment (yaswanthmyneni) [reply components/responsive-image/image.html:20]: done.

**Pull request merged**